packages/@aws-cdk/aws-dynamodb-global/README.md

```diff
@@ -0,0 +1,30 @@
+## @aws-cdk/aws-dynamodb-global
+
+Here is a minimal deployable Global DynamoDB tables definition:
+
+```typescript
+import globaldynamodb = require('@aws-cdk/aws-dynamodb-global');
+const CONSTRUCT_NAME = 'aws-cdk-dynamodb-global';
```

Comment:
These consts are not really adding much in terms of readability and they are only used once, just inline them